### PR TITLE
[IMP] im_livechat: add kanban view for session history

### PR DESCRIPTION
**Pull request by maad-odoo**
**Purpose of this commit:**
Introduce the Kanban view for live chat session history to enable 
easy navigation through sessions.

**task-4043438**

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Review by rapr-odoo — COMMENTED** [review]
Kanban view should be the default view instead of tree view. Apart from that looks good to me.

File: addons/im_livechat/views/discuss_channel_views.xml [hunk 1]
```diff
@@ -40,6 +40,33 @@
...
+            <field name="name">discuss.channel.kanban</field>
+            <field name="model">discuss.channel</field>
+            <field name="arch" type="xml">
+                <kanban js_class="im_livechat.discuss_channel_kanban" class="o_kanban_mobile" sample="1" quick_create="false">
+                    <templates>
+                        <t t-name="kanban-card">
+                            <div class="d-flex">
+                                <div class="d-flex flex-column">
+                                    <field class="fw-bolder" name="name"/>
+                                    <span class="fw-bold">Duration: <field class="d-inline fw-normal ms-1" name="duration" widget="float_time" options="{'displaySeconds': True}"/></span>
```
Comment (alexkuhn) [inline addons/im_livechat/views/discuss_channel_views.xml]: `ms-1` adds too much spacing, I removed it

Comment (alexkuhn) [reply addons/im_livechat/views/discuss_channel_views.xml]: I removed it

**Review by alexkuhn — APPROVED** [review]
LGTM

We could backport to 17.4 as suggested in task.
I'm already approving for master, so we are already set on this version!

@robodoo r+

**Comment by alexkuhn** [comment]
@robodoo r+

**Pull request closed without merging**